Multiple improvements

JustJoostNL opened the pull request:
- Improved mobile support.
- You can't click on cards anymore when the stream is not live.
- Improved header.
- Add footer.

JustJoostNL committed:
- Fix for base path error while testing in development environment.
- Improved mobile support.
- You can't click on cards anymore when the stream is not live.
- Improved header.
- Add footer.

LapsTimeOFF commented on next.config.js:
```diff
@@ -2,7 +2,6 @@
 const nextConfig = {
   reactStrictMode: true,
   output: "export",
-  basePath: "/",
```
No, this would break the integration in Github Pages

JustJoostNL replied:
How are you dealing with the following error then?

```
Error: Specified basePath /. basePath has to be either an empty string or a path prefix"
```

LapsTimeOFF replied:
Its only on development, you remove the line while you are in dev mode, then you put it back for prod

JustJoostNL replied:
Final solution should be this I guess:

```ts
basePath: process.env.NODE_ENV === "production" ? "/" : undefined
```

(Works great for me)

LapsTimeOFF commented on src/components/StreamCard.tsx:
```diff
@@ -139,6 +139,7 @@ const StreamCard: FC<Props> = ({ event, sessionKey, sessionDate }) => {
               ? `/player/vod/${event.video?.vod?.[sessionKey]}`
               : ``
           }
+          disabled={data ? data.status !== BroadcastStatus.BROADCASTING : true}
```
Breaks VOD

LapsTimeOFF commented on src/components/UpcomingEvents.tsx:
```diff
@@ -19,16 +19,31 @@ const UpcomingEvents: FC<Props> = ({ font }) => {
       <Typography
         variant="h2"
         sx={{ fontFamily: font, textAlign: "center", mb: 1 }}
+      >
```
This should stay in the main page, not this component

LapsTimeOFF commented on src/pages/index.tsx:
```diff
@@ -20,38 +20,68 @@ export default function Home() {
+          }}
         >
-          <b>Season 1: F1 Twitter Refugees league</b>
+          League Streaming Platform
```
Text not centered anymore

JustJoostNL replied:
That's on purpose in case we want an 3 dot menu at the left side, or some button next to the text. Also looks cleaner on mobile now.

LapsTimeOFF replied:
Yes, but it really dont looks nice on computer

LapsTimeOFF commented on src/pages/index.tsx:
```diff
@@ -20,38 +20,68 @@ export default function Home() {
       </Box>
 
       <UpcomingEvents font={inter} />
+      <Box
```
Divider would be nice

LapsTimeOFF commented on src/pages/index.tsx:
```diff
@@ -20,38 +20,68 @@ export default function Home() {
+          alignItems: "center",
+        }}
+      >
+        <Typography
```
Text should be a bit bigger and centered

LapsTimeOFF reviewed (changes requested):
Changes required, please check the messages

JustJoostNL committed:
- Fix base path issue with github pages.
- Fix breaking VOD because of cards disabled.
- Transfer season text to the main page.
- Center header text.
- Add divider between season text and upcoming events.
- Center copyright text and make text bigger.
